SU-SWS/stanford_capx
# Added capx_ssp and capx_webauth modules

For integration with single sign on modules.

To Test:
* Check out this branch on a sites site
* Enable the capx_webauth module
* Import your profile as a user entity
* Log in through webauth :D
